### DISCUSSION TITLE
How can I remove the yellow wavy line?

### DISCUSSION THREAD

**Discussion by cjr163** [redhat-developer/vscode-java#3729] [Q&A]
![无标题](https://github.com/user-attachments/assets/84d39f1b-a389-4eb6-b87b-47b09ef85d1e)
I took over an existing project, and IntelliJ IDEA does not show this warning, it makes all files yellow.

**Comment by cjr163** [comment]
I mean, I don't want to modify the original file. I need other warnings, but not this one.

**Comment by NoahELE** [comment]
Since it is a generic class, use parameterized constructor `new ResponseResult<>(...)`. The empty angle brackets allow java to deduct the type automatically.
